src/Pages/Settings/Addons.elm

```diff
@@ -65,12 +68,116 @@ view : Model -> Document Msg
+                        , settingsToggleBlock "idna" "Internationalized Domain Names for Python"
+                        , settingsToggleBlock "Python Image Library" ""
+                        , settingsToggleBlock "Python Crypto Library" ""
+                        , settingsToggleBlock "requests" "Requests is an elegant and simple HTTP library for Python, built for human beings."
+                        , settingsToggleBlock "six" "Six is a Python 2 and 3 compatibility library. It provides utility functions for smoothing over the differences between the Python versions with the goal of writing Python code that is compatible on both Python versions. See the documentation for more information on what is provided."
+                        , settingsToggleBlock "urllib3" "urllib3 is a powerful, user-friendly HTTP client for Python. Much of the Python ecosystem already uses urllib3 and you should too. urllib3 brings many critical features that are missing from the Python standard libraries"
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.service")
+                        , settingsToggleBlock "Version Check" "Kodi Version Check only supports a number of platforms/distros as releases may differ between them. For more information visit the kodi.tv website."
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.gui.skin")
+                        , settingsToggleBlock "Estouchy" "Skin designed to be used on touchscreen devices like tablets and smartphones"
+                        , settingsToggleBlock "Estuary" "Estuary is the default skin for Kodi 17.0 and above. It attempts to be easy for first time Kodi users to understand and use."
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.webinterface")
+                        , settingsToggleBlock "Kodi web interface - Chorus2" "Browse and interact with your Music, Movies, TV Shows and more via a web browser. Stream music and videos on your browser. Edit and manage your Kodi media library."
+
+                        , row[ width (px 800),Background.color Colors.headerBackground, paddingXY 20 25][
+                            Input.button[Background.color Colors.cerulean, Font.color Colors.white,paddingXY 30 8, Font.size 14, Border.rounded 2]{
+                              onPress = Nothing,
+                              label = text "SAVE"
+                              }
+                            ]
+                        , el[height (px 100)](text "")
+                        ]
+            ]
         ]
     }
+
+
+settingsToggleBlock : String -> String -> Element msg
+settingsToggleBlock title description =
```

Comment:
I would probably move this to it's own file, so that it's clear, that it's used from multiple frontends

Reply:
Yeah that way other blocks can also be added to that. Will do it.